include/SDL3_ttf/SDL_ttf.h

```diff
@@ -2326,6 +2326,95 @@ extern DECLSPEC int SDLCALL TTF_SetFontScriptName(TTF_Font *font, const char *sc
  */
 extern DECLSPEC SDL_bool TTF_IsFontScalable(const TTF_Font *font);
 
+/**
+ * Subpixel rendering mode
+ *
+ * \sa TTF_GetSubpixelMode
+ * \sa TTF_SetLcdGeometry
+ * \sa TTF_SetLcdFilter
+ * \sa TTF_SetLcdFilterWeights
+ */
+typedef enum
+{
+  TTF_SUBPIXEL_MODE_UNKNOWN = 0,            /* Unknown */
+  TTF_SUBPIXEL_MODE_HARMONY = 1,            /* Harmony LCD rendering */
+  TTF_SUBPIXEL_MODE_CLEARTYPE_STYLE = 2,    /* ClearType-style LCD rendering */
+} TTF_SubpixelMode;
+
+/**
+ * LCD filter, for ClearType-style LCD rendering
+ * mapping to FreeType FT_LcdFilter
+ */
+typedef enum
+{
+    TTF_LCD_FILTER_NONE    = 0,
+    TTF_LCD_FILTER_DEFAULT = 1,
+    TTF_LCD_FILTER_LIGHT   = 2,
+    TTF_LCD_FILTER_LEGACY1 = 3,
+    TTF_LCD_FILTER_LEGACY  = 4,
+} TTF_LcdFilter;
+
+/**
+ * Detect which subpixel rendering mode FreeType uses internally when LCD functions are used.
+ *
+ * \returns subpixel rendering mode
+ *
+ * \since This function is available since SDL_ttf 3.0.0
+ *
+ * \sa TTF_SetLcdGeometry
+ * \sa TTF_SetLcdFilter
+ * \sa TTF_SetLcdFilterWeights
+ */
+TTF_SubpixelMode SDLCALL TTF_GetSubpixelMode(void);
```

Comment:
Missing `extern DECLSPEC ` (same for the other new functions)

Sorry that I missed noting that before.